addons/mail/static/src/models/composer_suggestion.js

```diff
@@ -49,6 +67,12 @@ registerModel({
             inverse: 'mainSuggestions',
             readonly: true,
         }),
+        /**
+         * The text that identifies this suggestion in a mention.
+         */
+        mentionText: attr({
+            compute: '_computeMentionText',
+        }),
         record: one('Record', {
             compute: '_computeRecord',
         }),
```

Comment:
Do we actually still use this field for anything else than retrieving the localId of the record? Otherwise, we could get rid of one level of indirection by directly computing the recordLocalId.

Reply:
It's still used in template, and in some other code that hasn't been taken into account for the moment.
It's best to keep it for now, to keep PR focus on newly introduced field.
Removal of this field will possibly be done at a later date.